src/fpm/manifest/build.f90

```diff
@@ -52,20 +52,23 @@ module fpm_manifest_build
 
 
     !> Construct a new build configuration from a TOML data structure
-    subroutine new_build_config(self, table, error)
+    subroutine new_build_config(self, table, package_name, error)
 
         !> Instance of the build configuration
         type(build_config_t), intent(out) :: self
 
         !> Instance of the TOML data structure
         type(toml_table), intent(inout) :: table
 
+        !> Package name
+        character(len=*), intent(in) :: package_name
```

Comment:
Another option would be to inject the entire `package_config_t`. You can get the `build_config_t` from it, as well as the package name.

Reply:
right - along with a nice circular dependency ;)